test/dynamo/test_cudagraphs.py

```diff
@@ -106,6 +106,7 @@ def fn(x, y):
         fn(x, y)
 
     @patch("torch._functorch.config.use_functionalize", True)
+    @patch_all()
```

Comment:
Why this one needs `patch_all`?

Reply:
@malfet I think this is exactly the conflict that's showing up in the PR currently. @jataylo, if there's a reason we need the decorator to stay, please explain the reason here. Otherwise, let's resolve the conflict, and then re-request an approval from Nikita when CI is green.